### ISSUE TITLE
Some accessibility labels are not translated

### ISSUE THREAD

**Issue by Scrumplex** [telekom/scale#1205]
**Scale Version**
3.0.0-beta.111

**Framework and version**
No Framework

**Current Behavior**
Some accessibility labels are not translated to German, for pages with `<html lang="de" ...`

Affected components:
- scale-notification-message's close button has `aria-label="close"`
- scale-app-header's sector-navigation
  - active sector is labelled as `active` despite the language settings
![image](https://user-images.githubusercontent.com/11587657/189891652-b9b9b71e-97c6-4f85-8b7e-d13ff1bc6cc3.png)
  - additionally the sector navigration item has `aria-current="true"` which means there are two indicators that will be read by screen readers

**Code Reproduction**
```html
<scale-notification-message dismissible opened>
  Notification text goes here
</scale-notification-message>
```
```html
<scale-app-shell claim-lang="de" logo-title="App Name" logo-href="/" sector-navigation='[{"name":"Personal","id":"Personal38","href":"#personal"},{"name":"Business","id":"Business39","href":"#business"}]' active-sector-id="Personal38">
  <div style="height: 800px; padding: 48px 0;">
    Scroll to test header behavior
  </div>
  <div style="height: 800px; padding: 48px 0;">
    Scroll more to test header behavior
  </div>
</scale-app-shell>
```

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Microsoft Edge, Firefox
 - Version Firefox 104.0.2, Microsoft Edge 105.0.1343.33

**Smartphone (please complete the following information):**
N/A

**Additional context**

**Comment by felix-ico** [comment]
Hi @Scrumplex sorry for the very late reply. I think the accessibility labels need to be set by the consumer, and are not automatically derived by the lang attribute. The header you mention has also been deprecated, and the new telekom-header should have many improvements in regards to accessibility, so these issues should all be solved.
